testing/get-coursier.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# This is a script used in CI to install coursier
+set -euxo pipefail
+
+. /etc/lsb-release
+
+COURSIER_URL='https://github.com/coursier/coursier/releases/download/v2.0.0/cs-x86_64-pc-linux'
+COURSIER_HASH='e2e838b75bc71b16bcb77ce951ad65660c89bda7957c79a0628ec7146d35122f'
+
+check() {
+    echo "$COURSIER_HASH  $TGZ" | sha256sum --check
+}
+
+ARTIFACT="$HOME/.coursier/coursier"
+mkdir -p "$(dirname "$ARTIFACT")"
+if ! check >& /dev/null; then
+    rm -f "$ARTIFACT"
+    curl --location --silent --output "$ARTIFACT" "$COURSIER_URL"
+    check
+fi
```

Comment:
ah, this is old caching code, can probably be removed from both locations (it was only briefly used for the swift stuff but then it made the pipeline _slower_ rather than faster)